brainglobe_napari_io/napari.yaml

```diff
@@ -10,9 +10,9 @@ contributions:
     title: Brainreg Read Directory Atlas Space
     python_name: brainglobe_napari_io.brainreg.reader_dir_atlas_space:brainreg_read_dir_atlas_space
 
-  - id: brainglobe-napari-io.cellfinder_read_dir
-    title: Cellfinder Read Directory
-    python_name: brainglobe_napari_io.cellfinder.reader_dir:cellfinder_read_dir
+  - id: brainglobe-napari-io.workflows_wholebrain_cell_read_dir
```

Comment:
should this not be `brainmapper_read_dir` now?

Reply:
It should (everything is changing very quickly), but I've raised #59 to track this properly as it affects functionality too.